PWGEM/Dilepton/Core/Dilepton.h

```diff
@@ -822,23 +833,38 @@ struct Dilepton {
       float deta = v1.Eta() - v2.Eta();
       float dphi = v1.Phi() - v2.Phi();
       o2::math_utils::bringToPMPi(dphi);
+      float opAng = 0;
```

Comment:
can you use pairutil::getOpeningAngle? This is in PairUtilities.h.

Reply:
Thank you very much for pointing this out. I am sorry that I overlooked this.